pkg/tfbridge/schema.go

```diff
@@ -655,6 +655,15 @@ func (ctx *conversionContext) makeObjectTerraformInputs(
 		return nil, err
 	}
 
+	// Iterate over the TF schema and add an empty list for each list type not in the results.
+	tfs.Range(func(key string, value shim.Schema) bool {
+		schema := tfs.Get(key)
+		if schema.Type() == shim.TypeList && result[key] == nil {
```

Comment:
Does terraform distinguish between `nil` and missing?

Reply:
Great question but I doubt it given the way they use them: https://github.com/hashicorp/terraform-provider-aws/blob/3645bd03060df19f5cf29c132e26f7e40b56230f/internal/service/elbv2/listener.go#L425C1-L427C3

Reply:
In maps yes, in objects no, an object having an optional field F will have it nil if it's missing, there's no other way for it to have it missing.